pages/1.10/monitoring/logging/quickstart/index.md

```diff
@@ -93,7 +95,9 @@ You can access the Mesos stderr and stdout logs natively through the DC/OS CLI `
     Thu Dec 15 00:51:31 2016 ip-10-0-1-177.us-west-2.compute.internal Command Executor (Task: test-log.2fc56009-c25d-11e6-81b2-9a5d88789ccd) (Command: sh -c 'while true;d...') [7131] stdout
     ```
 
-# View the Mesos task and system logs
+
+# View the Mesos and DC/OS logs
+
 
 You can view logs from tasks or the host subsystem with the `dcos node log` command.
```

Comment:
This sentence should look like this:
> You can view logs of DC/OS components with the `dcos node log` command.
That would also be great to add a link to the command's page (https://docs.mesosphere.com/1.10/cli/command-reference/dcos-node/dcos-node-log/)